webpack-widget.js

```diff
@@ -102,37 +127,142 @@ const getLegacyWidgetBuildConfig = (config = {}) => {
 		})
 	}
 
+	const devDemo = 'demo_dev.json'
+	const devDemoPath = `${srcPath}${devDemo}`
+	if (isRunningDevServer && fs.existsSync(devDemoPath)) {
+		console.log(`===== USING ${devDemo} ====`)
+		copyList.push({
+			flatten: true,
+			from: devDemoPath,
+			to: `${outputPath}${devDemo}`,
+			force: true
+		})
+	}
+
+	return copyList
+}
+
+const getDefaultRules = () => ({
```

Comment:
New Feature: getDefaultRules can now be used to create a copy of the webpack rules used in getLegacyWidgetBuildConfig. This can be used to alter, remove, and add to the module loaders.